### ISSUE TITLE
Cross-References

### ISSUE THREAD

**Issue by lauwers** [oasis-tcs/tosca-specs#60]
The Word version of the spec included a large number of cross references between various sections. These cross-references did not get carried over when the document was converted to Markdown. Cross-references need to be introduced where appropriate.

**Comment by tliron** [comment]
Recommendation: let's insert explicit semantic names for anchors (instead of automatically generated anchors, which unfortunately include section numbers).

**Comment by pmjordan** [comment]
I think there are three tasks to be performed here. They are:

1. Modify existing cross reference to point to an anchor which does not include the paragraph number.
2. Add cross references to the tables detailing the syntax.
3. Add cross references across the whole document.


# Task1
This is low priority but should be done first as it is much easier to do before the other two tasks
Make sure any existing links use the format 
 ```
[Node type](#node-types) 
 ```
NOT 
 ```
[Node type](#6436-node-types)
 ```

# Task 2
This is highest priority
Starting at section 6, pick a heading which defines a key word, e.g. 6.4.3.6 Node Types. Use search and replace to change 
 ```
Node type
 ```
to a link 
 ```
[Node type](#node-types) 
 ```
throughout the document in places where it seems appropriate.
Repeat for the next heading until you get bored. 
It should become obvious which key word defintions have already been crosslinked by other contributors.

# Task 3
This is medium priority
Search for text suggesting that the original contained a cross refernce and correct it to use the correct format and target. e.g. search for 'see section'

**Comment by koppor** [comment]
- [ ] Fix `A detailed description of the node type definition grammar is provided in Section XXX.` (`XXX` is 7.1 Node Type)

**Comment by pmjordan** [comment]
@koppor I am unable to find the text 

> * A detailed description of the node type definition grammar is provided in Section XXX.

when viewing the file https://github.com/oasis-tcs/tosca-specs/blob/working/tosca_2_0/TOSCA-v2.0.md

**Comment by lauwers** [comment]
Fixed
